README.md

```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
+## Table of contents
+1. [Getting Started](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#getting-started)
+2. [Video Presentation](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#video-presentation)
+3. [Installation](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#installation)
+4. [Running](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#running)
+5. [Gameplay Tutorial](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#gameplay)
+6. [Discord Bot Installation Guide](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#discord-bot-installation-guide)
+7. [MongoDB Community Edition Installation Guide](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#mongodb-community-edition-installation-guide)
+8. [Credits](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#credits)
 
-1. Python 3.11 is [recommended](https://github.com/DisnakeDev/disnake/pull/1135#issuecomment-1847303628).
+
+
+## Getting Started
+
+1. Python 3.11 is [recommended](https://github.com/DisnakeDev/disnake/pull/1135#issuecomment-1847303628). Python 3.10 - 3.12 will probably work.
 2. Setup a [Discord Bot](https://docs.disnake.dev/en/stable/discord.html). <!-- TODO: explain better once we have an idea on how the bot works (e.g. what permissions are required) -->
-3. clone the repository:
+3. Clone the repository :
    ```shell
    git clone https://github.com/SincereSingularities/SincereSingularities/
    cd SincereSingularities
    pip install -e .
    ```
-4. Set the `BOT_TOKEN` environment variable to your Token using the `.env` file.
-5. Run The Game:
+4. Setup and run [MongoDB Community Edition](https://www.mongodb.com/docs/manual/administration/install-community/)
+5. Set the `BOT_TOKEN`, `DB_HOST`, `DB_PORT` and `DB_NAME` environment variables using the `.env` file.
+6. Run The Game:
    ```shell
    python -m sincere_singularities
    ```
-6. Play!
+7. Play!
+
+## Video Presentation
+
+Disclaimer: `The video contains Spoilers`
+
+<details>
+    <summary>Video Presentation / Description</summary>
+    https://www.youtube.com/
+</details>
+
+
+## Installation
+<details>
+    <summary>Detailed Installation Guide ⚙️</summary>
+
+### 1. Requirements:
+   1. [Python 3.11](https://www.python.org/downloads/release/python-3110/)
+   2. [MongoDB Community Edition](https://www.mongodb.com/docs/manual/administration/install-community/). See [MongoDB Installation Guide]
+   3. [Discord Bot](https://docs.disnake.dev/en/stable/discord.html). See [Discord Bot SetUp Guide]
+### 2. Download:
+Run this command in the directory you want to download it to.
+   ```shell
+   git clone https://github.com/SincereSingularities/SincereSingularities/
+   cd SincereSingularities
+   ```
+### 3. Install the Game as a PIP package & install requirements:
```

Comment:
pip already installs the requirements. Either remove the reference or make it clear that pip already does that.